cedar-rust-hello-world/Cargo.toml

```diff
@@ -9,8 +9,7 @@ publish = false
 serde_json = "1.0"
 
 [dependencies.cedar-policy]
-version = "4.0.0"
+version = "4.1.0"
 git = "https://github.com/cedar-policy/cedar"
 branch = "main"
```

Comment:
Maybe `release/4.1.x`?

Reply:
I was spooked by the line 15 warning not to change this haha

Reply:
+1 to update branch

Reply:
Updated